README.md

```diff
@@ -36,64 +72,123 @@ To successfully receive a payment, these steps should be implemented:
 
 3. The customer returns, and should be satisfied to see that the order was paid and is now being processed.
 
-## Getting started ##
+## Importing the client
```

Comment:
Why use writeHead with 302? just use the standard res.redirect included by express :)